### DISCUSSION TITLE
Could not compile `accesskit_macos` 0.15.0

### DISCUSSION THREAD

**Discussion by Perrelli9338** [bevyengine/bevy#14805] [Q&A] [answered]
Hi there!
I'm using Bevy **0.14.1** and when I try to build my game for macOS with GitHub Workflow it fails as the title explain, I haven't found issues about this. Any solution?

**Comment by unknownue** [comment, marked as answer]
Are you using nightly rust ? Downgrade rust version to 2024-06-10 fix this on my machine.

**Reply by Perrelli9338** [reply]
Yes, that was the problem. 
Using the `stable` channel of toolchain now compiles.
Thank you :)

**Comment by Salzian** [comment]
Through trial and error I found out that `nightly-2024-08-14` is the latest working version.

With `2024-06-10` I ran into the problem of `cargo` missing. Apparently not all `nightly` releases have all components included.
